src/components/ContentProfile/styled.ts

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
```

Comment:
Do we need an extra Wrapper? Shouldn't one style (combined Wrapper and WrapperCenter) be able to manage all centering and sizing on it's own?

Reply:
For the first time, it didn't work for me. Now I managed to merge with a small adjustment of styles. If you have ideas on how to make it smaller, I'll be happy. 👍

Reply:
I just played with it and it was the `width`. Especially 75% max-width made a gap on the right side. 
```

export const Wrapper = styled.div`
	display: grid;
	grid-template-columns: 1fr 3fr 1fr 1fr;
	gap: 2em;
	justify-items: center;
	@media (max-width: 50em) {
		grid-template-columns: 1fr 5fr 1fr;
		grid-template-rows: auto auto;
		position: relative;
		top: 12em;
	}
`

export const WrapperLabel = styled.div`
	grid-column: 3;
	text-align: center;
	@media (max-width: 50em) {
		grid-column: 2;
		grid-row: 1;
	}
`

export const WrapperInformation = styled.div`
	grid-column: 2;
	margin-bottom: 3em;
	& h2 {
		color: ${({ theme }) => theme.text.secondary};
	}
	@media (max-width: 50em) {
		grid-column: 2;
		grid-row: 2;
	}
`
```

And for the page itself this: 

```
import {
	Wrapper,
	WrapperInformation,
	StyledIcon,
	StyledButton,
	WrapperLabel,
} from './styled'

                       <Wrapper>
				<WrapperInformation>
					<Switch>
						<Route exact path="/my-profile">

	                        </WrapperInformation>
				<WrapperLabel>
					<h5>Vyber si štítek:</h5>
					<StyledButton>
						<Button subtitle="Mentor" />
						<Button subtitle="Buddy" />
						<Button subtitle="Mentee" />
					</StyledButton>
				</WrapperLabel>
			</Wrapper>
```

That means we only get one grid, one wrapper and two columns (one for WrapperInformation and one for WrapperLabel). WrapperCenter and WrapperContent can completely disappear and functionality remains the same.

I would consider renaming WrapperInformation to just Information or something like that and similarly for WrapperLabel. :thinking:

Reply:
(Mentioning you because github keeps hiding this even when it's not resolved. :sweat_smile: @MartinaVilimova )

Reply:
Thank you for idea 👍